docs/topics/writing_templates.md

```diff
@@ -151,7 +151,7 @@ Making embedded media resizable is also possible, but typically requires custom
 
 ### `pageurl`
 
-Takes a Page object and returns a relative URL (`/foo/bar/`) if within the same Site as the current page, or absolute (`http://example.com/foo/bar/`) if not.
+`pageurl` takes a Page object and returns a relative URL (`/foo/bar/`) if the Page object is within the same Site as the current page. If the Page object is not within the same Site as the current page, `pageurl` will return an absolute URL like: (`http://example.com/foo/bar/`).
```

Comment:
The existing content seems fine to me. It’s not that useful to restate this is about `pageurl` since that’s the heading right before, and I don’t think making this content 2x the length improves clarity much.

```suggestion
Takes a Page object and returns a relative URL (`/foo/bar/`) if within the same Site as the current page, or absolute (`http://example.com/foo/bar/`) if not.
```